From ba47e9abbf355509aacaa14d4b904afd6c9e26a5 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Mon, 15 Jun 2015 19:35:23 +0800
Subject: [PATCH] Revert "MLK-11080-1 dts: imx7d-sdb: add OOB GPIO config for
 wlan node"

WL_HOST_WAKEUP feature causes system suspend/resume break, so revert.
This reverts commit ce9e893341c6196a5c31b3265525218c537d3e3e.
---
 arch/arm/boot/dts/imx7d-sdb.dts | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 81520f6..1d8fd74 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -99,15 +99,12 @@
 			regulator-max-microvolt = <5000000>;
 			regulator-name = "wlreg_on";
 			gpio = <&gpio4 21 GPIO_ACTIVE_HIGH>;
-			startup-delay-us = <100>; /* WL_REG_ON needs 61 usec delay */
 			enable-active-high;
 		};
 	};
 
 	bcmdhd_wlan_0: bcmdhd_wlan@0 {
 		compatible = "android,bcmdhd_wlan";
-		/* OOB_IRQ: ECSPI2_SCLK (gpio4 20) */
-		gpios = <&gpio4 20 0>;
 		wlreg_on-supply = <&wlreg_on>;
 	};
 
@@ -812,8 +809,7 @@
 				MX7D_PAD_SD2_DATA1__SD2_DATA1   0x59
 				MX7D_PAD_SD2_DATA2__SD2_DATA2   0x59
 				MX7D_PAD_SD2_DATA3__SD2_DATA3   0x59
-				MX7D_PAD_ECSPI2_MOSI__GPIO4_IO21	0x19 /* WL_REG_ON */
-				MX7D_PAD_ECSPI2_SCLK__GPIO4_IO20	0x19 /* WL OOB */
+				MX7D_PAD_ECSPI2_MOSI__GPIO4_IO21	0x59 /* WL_REG_ON */
 			>;
 		};
 
-- 
1.8.0

